meson.build

```diff
@@ -506,9 +506,7 @@ if sys_windows
     'ecore_win32',
     'ecore_wl2',
     'ecore_x',
-    'ector',
     'edje',
-    'eet',
```

Comment:
[ector doesn't need eet](https://github.com/expertisesolutions/efl/blob/devs/joaoantoniocardoso/ector/redone/meson.build#L456) so you don't need to make um PR dependent on the other.